src/promptflow/promptflow/orchestrator/_chat_group_orchestrator.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional, List, Dict, Mapping, Any
+from pathlib import Path
+from promptflow.contracts.flow import ChatGroupRole
+from promptflow._constants import LANGUAGE_KEY, FlowLanguage
+from promptflow._utils.yaml_utils import load_yaml
+from promptflow.batch._base_executor_proxy import APIBasedExecutorProxy
+from promptflow.executor._result import LineResult
+
+class ChatGroupOrchestrator:
+    def __init__(
+        self,
+        chat_group_roles: List[ChatGroupRole],
+        run_id: Optional[str] = None,
+        max_turn: Optional[int] = None,
+        **kwargs
+    ):
+        self._run_id = run_id
```

Comment:
Do we need this run id?

Reply:
I think we need it since all the flow runs share this run id.

Reply:
Removed it since run id is passed in schedule_runs function